src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportGenerator;
+import java.util.stream.Collectors;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport() {
+        String headers = "fruit, amount" + "\n";
+        return headers + Storage.getAll().entrySet().stream()
+                .map(entry -> entry.getKey() + "," + entry.getValue())
+                .collect(Collectors.joining("\n"));
```

Comment:
add constants for header and separator. Also, it is better to use System.lineseparator()